Docker-Swarm-deployment/single-node/consent-validator/README.md

```diff
@@ -0,0 +1,43 @@
+# Introduction
+Docker swarm stack for IUDX Consent-Validation Deployment
+
+# Installation of Consent Validation
+## Create secret files
+1. Make a copy of sample secrets directory.
+
+```console
+ cp -r example-secrets/secrets .
+```
+2. Substitute appropriate values using commands whatever mentioned in config files. Configure the secrets/.consent.env file with appropriate values 
+3. Secrets directory after generation of secret files
```

Comment:
write command to generate keystore and update secret directory structure to include keystore.jks

Reply:
see https://github.com/datakaveri/iudx-deployment/tree/master/Docker-Swarm-deployment/single-node/auth#create-secret-files